guest-test/guest.test_executor.sh

```diff
@@ -8,13 +8,20 @@
 
 
 # @desc This script prepare and run $TESTCASE in Guest VM
+# based on $FEATURE selection
 
 ###################### Variables ######################
-SCRIPT_DIR="$( cd "$( dirname "$0" )" && pwd )"
-echo "$SCRIPT_DIR"
+# exec only if script being executed
+if [ "${BASH_SOURCE[0]}" -ef "$0" ]; then
+  SCRIPT_DIR="$( cd "$( dirname "$0" )" && pwd )"
```

Comment:
Hi, are the quotation marks really necessary? Or is it for coding style checking? 
Why not `SCRIPT_DIR=$( cd $( dirname "$0" ) && pwd )`   ?